int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -418,39 +431,45 @@ function edqSetGlobalIntuitiveConfiguration() {
 	];
 }
 /**
-* Pro Web - Address (Verification Engine)
-* Sets the configuration to use Pro Web - Address (Verification Engine)
-*/
+ * Pro Web Address Verification callback
+ */
 function edqValidateAddressCallBack() {
 	var edqProWebResponse = document.querySelector("#form-submit");
 	if (edqProWebResponse.hasAttribute("edq-metadata")) {
 		var edqProWebMetaDataJSON = JSON.parse(edqProWebResponse.getAttribute("edq-metadata"));
 		document.querySelector("#form-submit").style.display = "none";
-		edqStateLineSelector.value = edqProWebMetaDataJSON["State code"];
+		edqStateLineSelector.value = edqProWebMetaDataJSON["address.province"];
 	}
 	if (edqProWebCallbackValidation) {
 		if (edqProWebExecuteTransitionCallBack(edqProWebMetaDataJSON)) {
 			edqCheckoutPageWorkflows();
 		}
 	} else { edqCheckoutPageWorkflows(); }
 }
+/**
+ * Manage the button settings for the Button "form-submit", depending on the touchpoint.
+ */
 function edqCheckoutPageWorkflows() {
-	if (edqCurrentSubmitButtonSelectorIndex) {
-		buttonSelector[edqCurrentSubmitButtonSelectorIndex].style.display = "inline-block";
-		buttonSelector[edqCurrentSubmitButtonSelectorIndex].click();
+	if (edqCurrentSubmitButtonIndex) {
+		buttonSelector[edqCurrentSubmitButtonIndex].style.display = "inline-block";
+		buttonSelector[edqCurrentSubmitButtonIndex].click();
 	} else {
-		edqCurrentSubmitButtonSelector.style.display = "inline-block";
-		edqCurrentSubmitButtonSelector.click();
+		edqCurrentSubmitButton.style.display = "inline-block";
+		edqCurrentSubmitButton.click();
 	}
 }
+/**
+* Pro Web - Address (Verification Engine)
+* Sets the configuration to use Pro Web - Address (Verification Engine)
+*/
 function edqSetProWebConfiguration() {
 	/** This is intended to hide the form button in initial load of the page; just to show verification engine button in the form. **/
-	if (edqCurrentSubmitButtonSelector) {
-		edqCurrentSubmitButtonSelector.style.display = "none";
+	if (edqCurrentSubmitButton) {
+		edqCurrentSubmitButton.style.display = "none";
```

Comment:
It might be worth seeing if there's a CSS style that you can add to the button that will hide it, as opposed to using inline styles.

Reply:
I'll search for a way to do it from a CSS; let me know if we'll be doing it for this PR or in the code refactoring.

Reply:
We'll do that in a separate pull request.